client/me/connected-application-item/index.jsx

```diff
@@ -149,7 +149,7 @@ class ConnectedApplicationItem extends React.Component {
 							),
 						},
 						args: {
-							date: this.props.moment( authorized ).format( 'MMM D, YYYY @ h:mm a' ),
+							date: this.props.moment( authorized ).format( 'LL @ LT' ),
```

Comment:
Should it be `ll @ LT` instead, because `MMM` would display only the first three letters of the month?